nkms/crypto/keyring.py

```diff
@@ -109,12 +109,12 @@ def sign(self, message):
         """
         Signs a message and returns a signature with the keccak hash.
 
-        :param bytes message: Message to sign in bytes
+        :param Iterable message: Message to sign in an iterable of bytes
```

Comment:
Wait, can't it still be `bytes`?

Reply:
It can, but I just thought to make it clear that it can be an iterable of `bytes` objects.